.github/workflows/all_ubuntu.yml

```diff
@@ -26,3 +26,20 @@ jobs:
         run:  make -j6
       - name: Run all make 
         run: make -j6 check
+
+  sdfmt:
+    name: Build sdfmt via dub
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        dc: [dmd-latest, ldc-latest, dmd-2.100.2]

```

Comment:
I've added DMD v2.100.2 because it's more or less the version we currently use at Symmetry.

Reply:
Isn't there a way to tell dub to build everything? Selecting things one by one is how you end up with hole in the test plan.

Reply:
You have 4 subpackages there; AFAIK, this requires 4 cmdlines. And I guess the others would need some extra apt packages.

Reply:
none require that at the moment. I'll merge, but this seems like a problem dub has to solve.